google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/flexible/AppEngineFlexibleDeploymentEditor.java

```diff
@@ -186,14 +194,22 @@ private void resetRuntimeDisplay() {
     }
   }
 
+  private void updateStagedArtifactNameEmptyText() {
+    Path artifactPath = Paths.get(Strings.nullToEmpty(deploymentSource.getFilePath()));
+    AppEngineFlexibleDeploymentArtifactType deploymentArtifactType =
+        AppEngineFlexibleDeploymentArtifactType.typeForPath(artifactPath);
+    stagedArtifactNameTextField
+        .getEmptyText()
+        .setText(deploymentArtifactType.getDefaultArtifactName());
```

Comment:
No, but I'm not sure how tell that a run configuration is "new". IDEA manages the lifecycle of these config objects and from a quick debug session, it looks like `AppEngineDeploymentConfigurator.createDefaultConfiguration(DeploymentSource)` is called repeatedly while the edit configuration window is open, even for existing configurations. I don't know when we'd have the chance to default this only for new configurations and not existing ones.

The user will have to review this field anyways and considering they are using a custom runtime with their own Dockerfile, they should know that they need to fill this out. WDYT?